index.html

```diff
@@ -0,0 +1,109 @@
+<li class="desktop-link"><a href="#" class="nav">CC Compaign</a></li>
+</ul>
+    <div class="menu-icon">
+        <div class="item-closed"><a href="#" class="closing"><i class="fa fa-times" aria-hidden="true"></i></a>
+        </div>
+        <div class="mobile-nav"><a href="./index.html" class="closing">Home</a></div>
+        <div class="mobile-nav"><a href="./about.html" class="closing">About</a></div>
+        <div class="mobile-nav"><a href="#" class="closing">Join</a></div>
+      </div>
+<div class="item-menu"><a href="#" class="openMenu"><i class="fa fa-bars"></i></a></div>
+</nav>
+</header>
+<section class="home-page">
+<p class="heading">"Hello! Sharing World"</p>
+<h1 class="big-heading">Creative Commons Global Summit 2015</h1>
+<p class="para">A joyful celebration believing in the value of openness and sharing, creating 
+a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+<p class="date">2015.10.15(Thu) ~ 16(Fri)</p>
+<p class="location">@ National Museum of Korea, Art Center Nabi and more</p>
+</section>
+<section class="about">
+    <h2 class="about-section">Main Program</h2>
+    <hr>
+    <div class="about-container">
+        <div class="list-item">
+           <div class="image"><img src="./images/diversity_2_FILL0_wght400_GRAD0_opsz48.svg" alt="img"></div>  
+            <h3>Lecture</h3>
+            <p class="graph">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+        </div>
+        <div class="list-item">
+            <div class="image"><img src="./images/desktop_windows_FILL0_wght400_GRAD0_opsz48.svg" alt="img"></div>
+            <h3>CC Exhibition</h3>
+            <p class="graph">Appreciate various creations applying CC license of artists, organized from Art Center Nabi.</p>
+        </div>
+        <div class="list-item">
+            <div class="image"><img src="./images/forum_FILL0_wght400_GRAD0_opsz48.svg" alt="img"></div>
+            <h3>Forum</h3>
+            <p class="graph">Have the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="list-item">
+            <div class="image"><img src="./images/home_repair_service_FILL0_wght400_GRAD0_opsz48.svg" alt="img"></div>
+            <h3>Workshop</h3>
+            <p class="graph">Try creating your own work using open source license rather than just watching at it.</p>
+        </div>
+        <div class="list-item">
+            <div class="image"><img src="./images/hub_FILL0_wght400_GRAD0_opsz48.svg" alt="img"></div>
+            <h3>CC Ignite</h3>
+            <p class="graph">Get opportunities to network with CC affiliates around the world, also after the summit.</p>
+        </div>
+    </div>
+    <div class="end"><u>See the whole Program</u></div>
+    
+    <button class="button">Join CC Global Summit 2015</button>
+</section>
+<section class="featured" id="featured-page"></section>
```

Comment:
- [ ] I would like to recommend you work on all your section and page mobile design. For example the contributor section, there are a few differences between your design and the original one. Please fix the design for all your mobile version for all sections and page:+1: